temporian/core/operators/filter.py

```diff
@@ -105,20 +105,7 @@ def before(
     assert isinstance(input, EventSetNode)
 
     if isinstance(timestamp, datetime):
-        if not input.schema.is_unix_timestamp:
```

Comment:
Should we have a warning here?

I'm in favor of removing the exception to make the operation more flexible but I'm wondering if we should warn the user about this.

Reply:
Good point.

*Print warning instead of error.
*Add `is_unix_timestamp` setter.